docs/development/manual-html-operations.md

```
@@ -285,9 +285,20 @@ SELECT COALESCE(url,'') <> '' AS has_doc, COALESCE(manual_url,'') <> '' AS has_s
 
 タスク一覧のR/S列にVLOOKUPの数式が入っていない場合は、`SeeFTメニュー` から `fillManualUrlFormulas` を実行する。4行目から最終行+50行まで数式を貼るので、行が増えても追従する。
 
-引けているかを `inspectManualUrlLookup` で確認する。対応表の中身と、タスク一覧でM列が埋まっている行のR/S列が表示される。
+貼ったら `SeeFTメニュー > マスタをSeeFTに送信する > 【事前確認】マニュアルURLの紐付けを点検する` を実行する。対応表とM列のずれを送信前にすべて列挙する（issue #495）。
 
-確認できたら `SeeFTメニュー > タスク送信` を実行する。これでDBの `tasks.manual_url` が更新され、mobileのシフトカードに「HTML版」ボタンが出る。
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)